resources/to_timestamp.py

```diff
@@ -0,0 +1,20 @@
+"""Converts datetime objects to unix time."""
+# A module with a couple simple functions I got tired of writing/finding
+import datetime
+import logging
+
+
+def to_timestamp(datetime_object):
+  logging.debug('datetime: ' + str(datetime_object))
```

Comment:
same comment here you ususally don t need a method for that:
time.mktime(dt.timetuple())